.kitchen.docker.yml

```diff
@@ -0,0 +1,11 @@
+---
+driver:
+  name: docker
+  use_sudo: false
+  volume: /var/run/docker.sock:/var/run/docker.sock
+
+platforms:
+- name: ubuntu-14.04
```

Comment:
We mention 14.04 here, but 16.04 lower down (in `spec/spec_helper.rb`) and then trusty (14.04) in the `.travis.yml`. Which one should it be? Should we upgrade to 16.04 everywhere?

Reply:
Good point! I think we should upgrade to `16.04` yes.